ang/civicase/Actions--print-merge.js

```diff
@@ -0,0 +1,36 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.service('PrintMergeCaseAction', PrintMergeCaseAction);
+
+  function PrintMergeCaseAction () {
+    /**
+     * Get path object to print/merge
+     *
+     * @param {Array} cases
+     * @return {Object}
+     */
+    this.getPath = function (cases) {
+      var contactIds = [];
+      var caseIds = [];
+
+      _.each(cases, function (item) {
+        caseIds.push(item.id);
+        contactIds.push(item.client[0].contact_id);
+      });
+
+      var popupPath = {
```

Comment:
Just wondering. Can't we just return the object directly, without saving it in a var popupPath?
```js
return {
  path: 'civicrm/activity/pdf/add',
  query: {
    action: 'add',
    reset: 1,
    context: 'standalone',
    cid: contactIds.join(),
    caseid: caseIds.join()
  }
};
```
In this way we save an extra variable declaration and put the main focus on the logic and not extra piece of code.
And we can make up for readability of the function by adding extra line to function docs like this.
```js
    /** 
     * Get path object to print/merge the activity
     *
     * @param {Array} cases
     * @return {Object} popup path object
     */